backend/server.js

```diff
@@ -20,7 +31,36 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hi there Allan!')
+})
+
+// Sign up endpoint
+app.post('/signup', async (req, res) => {
```

Comment:
To make your API more RESTful, make sure you name endpoints with pluralized nouns. This could be called `/users`, because it's a new user you're creating. The login endpoint could be called `/sessions` or similar, because you're creating a logged in session for the user